webrtc/libjingle/xmpp/boshsocket.cc

```diff
@@ -155,7 +156,15 @@ namespace buzz {
 #ifdef FEATURE_ENABLE_SSL
     if (tls_ != buzz::TLS_DISABLED) {
       first_socket = rtc::SSLAdapter::Create(first_socket);
+// For non-production builds, relax certificate requirements. This will allow the use
+// of a debugging proxy, for example.
+#ifndef _PRODUCT
```

Comment:
Oh wait, I skipped a step in my thinking about this (and this might not
even be catching). Let me at least fix *that* for this PR ;-)

On Fri, Sep 18, 2015 at 1:41 PM, Will Kelleher <notifications@github.com>
wrote:

> In webrtc/libjingle/xmpp/boshsocket.cc
> <https://github.com/PersonifyInc/webrtc/pull/13#discussion_r39887372>:
>
> > @@ -155,7 +156,15 @@ namespace buzz {
> >  #ifdef FEATURE_ENABLE_SSL
> >      if (tls_ != buzz::TLS_DISABLED) {
> >        first_socket = rtc::SSLAdapter::Create(first_socket);
> > +// For non-production builds, relax certificate requirements. This will allow the use
> > +// of a debugging proxy, for example.
> > +#ifndef _PRODUCT
>
> I don't think it needs to be with this PR. I think we just need to change
> the StagePresence gclient invocation to potentially set this in the webrtc
> build preprocessor settings if it is set in our CMake config. Not exactly
> sure what the steps for that are.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PersonifyInc/webrtc/pull/13/files#r39887372>.
>